common/persistence/sql/sqlplugin/postgres/plugin.go

```diff
@@ -91,3 +98,8 @@ func (d *plugin) createDBConnection(cfg *config.SQL) (*sqlx.DB, error) {
 	db.MapperFunc(strcase.ToSnake)
 	return db, nil
 }
+
+// TODO: implement postgres specific support for TLS
+func registerTLSConfig(cfg *config.SQL) error {
+	return fmt.Errorf("tls for postgres has not been implemented")
```

Comment:
nit: why using Errorf? Probably better to statically define this as a const in the file and return it here.  We have similar pattern in other parts of the code.